deploy/modules/gcp/tidb-cluster/values/default.yaml

```diff
@@ -0,0 +1,22 @@
+# Basic customization for tidb-cluster chart that is suitable for a GKE cluster
+timezone: UTC
+
+pd:
+  storageClassName: pd-ssd
+tikv:
+  storageClassName: local-storage
+tidb:
+  service:
+    type: LoadBalancer
+    annotations:
+      cloud.google.com/load-balancer-type: "Internal"
+  separateSlowLog: true
+monitor:
```

Comment:
The monitor configuration is changed, there is a tidb-monitor-initializer which is the same version as of tidb.

Reply:
do we need a separate file for GCP? I would like to re-use files and use kustomize if needed.

Reply:
@gregwebs The main difference between GCP and AWS is the storage class and load balancer annotations. We probably don't need kustomize, we can have a common default values.yaml and then `set` values as needed in the `helm_release` resource.

@tennix I don't think these values are conflicting with the configuration for the initializer. Is there anything that needs to be added/changed that I am missing?

Reply:
In the case of annotations you should also have the option of adding them for both GCP and AWS.